src/components/DeckMechanisms/index.js

```diff
@@ -358,8 +359,12 @@ export default class DeckMechanisms extends React.Component {
           'T' + arrayRandom([1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 13, 14])
         const token = resolveCardForLevel({ id })
         token.level = [5, 6, 6, 8, 10][card.level - 1]
+        token.rarity = null
```

Comment:
This has been done on master with https://github.com/KittySparkles/stormbound-kitty/commit/e7fec23415390409b8ab434c5eefe4c51c309a99.